client/src/carousel.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import styled from 'styled-components';
+
+// === STYLES === //
+
+const Wrapper = styled.div`
+  width: 1280px;
+  overflow: hidden;
+`;
+
+const CarouselContainer = styled.div`
+  display: flex;
```

Comment:
I like the use of flex here. Also, very good and clean code overall.

Reply:
Thanks!